pkg/operation/botanist/component/resourcemanager/resource_manager.go

```diff
@@ -1772,14 +1772,16 @@ var (
 	// TimeoutWaitForDeployment is the timeout used while waiting for the Deployments to become healthy
 	// or deleted.
 	TimeoutWaitForDeployment = 5 * time.Minute
+	// WaitForDeployment is an alias for retry.Until. Exposed for tests.
+	WaitForDeployment = retry.Until
```

Comment:
This alias should rather be called something like `Until`, as it doesn't do anything with deployments in particular